fab/src/main/res/drawable/fab_round_shape.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape android:shape="oval" xmlns:android="http://schemas.android.com/apk/res/android">
+    <solid android:color="#fff"/>
+</shape>
```

Comment:
Please use 6- or 8-digit colour codes to keep consistency with the rest of the project.